diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -789,3 +789,12 @@ private fun ASTNode.calculateLineNumber() = getRootNode()
         require(it >= 0) { "Cannot calculate line number correctly, node's offset $startOffset is greater than file length ${getRootNode().textLength}" }
         it + 1
     }
+
+/**
+ * @return the number of lines in a block of code.
```

Comment:
```suggestion
 * Count number of lines in code block. Note: only *copy* of a node should be passed to this method, because the method changes the node.
 * @return the number of lines in a block of code.
```